### DISCUSSION TITLE
How is this plugin different from the default auto idle hehaviour?

### DISCUSSION THREAD

**Discussion by RePacked** [RoguedBear/BetterDiscordPlugin-AutoIdleOnAFK#38] [Q&A]
Discord puts your status to idle after some time by default. So what exactly does this plugin do differently?

**Comment by RoguedBear** [comment]
when the plugin was released, discord did not have that feature enabled for majority of users. I only saw forum posts about something like that existing but never in action.
Now it appears discord has rolled out this feature for alot of users, and i saw it in action on my friend's laptop too.

there are many other features planned to be added and are labelled under the [enhancement tag](https://github.com/RoguedBear/BetterDiscordPlugin-AutoIdleOnAFK/issues?q=is%3Aissue+is%3Aopen+label%3Aenhancement) in issues.

though those issues date back to the beginning of the year, i still havent dropped the plan to implement them. I'll get around to doing that when i get free time during the holidays

**Reply by Strana** [reply]
Discord's auto-idle thing seems to activate after around 10 minutes of inactivity, so your plugin may still be useful if someone wants to their status to go idle sooner.

**Reply by RoguedBear** [reply]
> Discord's auto-idle thing seems to activate after around 10 minutes of inactivity, so your plugin may still be useful if someone wants to their status to go idle sooner.

thanks for pointing that out! If someone wants to go AFK before 10 minutes, the plugin does allow a minimum of 5 minutes before you go Idle. and it could even be lower than that by modifying the config file ;)

**Reply by Chirunos** [reply]
> > Discord's auto-idle thing seems to activate after around 10 minutes of inactivity, so your plugin may still be useful if someone wants to their status to go idle sooner.
> 
> thanks for pointing that out! If someone wants to go AFK before 10 minutes, the plugin does allow a minimum of 5 minutes before you go Idle. and it could even be lower than that by modifying the config file ;)

*You have to first change the timeout once for the setting to appear.

And does it work for 1 minute?

**Reply by RoguedBear** [reply]
@Gersonzao  yes you can set an AFK timeout 1 minute but that cannot be done via the plugin's settings page. You can modify the plugin's config file to change the timeout to any number. I have documented the steps to do that in my comment here:  https://github.com/RoguedBear/BetterDiscordPlugin-AutoIdleOnAFK/issues/2#issuecomment-1024280678
